.github/CODEOWNERS

```diff
@@ -0,0 +1,16 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# More details are here: https://help.github.com/articles/about-codeowners/
+
+# The '*' pattern is global owners.
+
+# Order is important. The last matching pattern has the most precedence.
+# The folders are ordered as follows:
+
+# In each subsection folders are ordered first by depth, then alphabetically.
+# This should make it easy to add new rules without breaking existing ones.
+
+# Global:
+
+* @cameel @r0qs
```

Comment:
what about you @0xisk? :)

I mean, at this point it's probably hard to tell who owns what because we don't even know the final dir structure.